src/components/Nodes/LineGraphNode/LineGraphNode.tsx

```diff
@@ -0,0 +1,236 @@
+    },
+    []
+  )
+
+  const handleLegendYChange = useCallback(
+    (e: ChangeEvent<HTMLInputElement>) => {
+      setEditableLegendY(e.target.value)
+    },
+    []
+  )
+
+  const handleLegendVisibleChange = useCallback(() => {
+    setLegendVisible(!legendVisible)
+  }, [legendVisible])
+
+  return (
+    <GraphNodeWrapper
+      title={title}
+      onTitleSave={onTitleSave}
+      onDescriptionSave={onDescriptionSave}
+      description={description}
+      editable={editable}
+    >
+      <div className={styles.container}>
+        <div
+          className={classNames(styles.graph, {
+            [styles.active]: editActive,
+          })}
+        >
+          <Line
+            data={
+              editable
+                ? Object.values(graphData).map(([key, value]) => ({
+                    id: key,
+                    data: Object.entries(value)
+                      .filter(
+                        ([, values]) => values[0] !== '' && values[1] !== ''
+                      )
+                      .map(([, values]) => ({
+                        x: values[0],
+                        y: values[1],
+                      })),
+                  }))
+                : Object.entries(data).map(([key, value]) => ({
+                    id: key,
+                    data: Object.entries(value).map(([, values]) => ({
+                      x: values[0],
+                      y: values[1],
+                    })),
+                  }))
+            }
+            {...config}
+            axisBottom={{
+              tickSize: 5,
+              tickPadding: 5,
+              tickRotation: 0,
+              legend: editableLegendX,
+              legendOffset: 36,
+              legendPosition: 'middle',
+            }}
+            axisLeft={{
+              tickSize: 5,
+              tickPadding: 5,
+              tickRotation: 0,
+              legend: editableLegendY,
+              legendOffset: -40,
+              legendPosition: 'middle',
+            }}
+            legends={
+              legendVisible
+                ? [
+                    {
+                      anchor: 'bottom-right',
+                      direction: 'column',
+                      justify: false,
+                      translateX: 100,
+                      translateY: 0,
+                      itemsSpacing: 0,
+                      itemDirection: 'left-to-right',
+                      itemWidth: 80,
+                      itemHeight: 20,
+                      itemOpacity: 0.75,
+                      symbolSize: 12,
+                      symbolShape: 'circle',
+                      symbolBorderColor: 'rgba(0, 0, 0, .5)',
+                      effects: [
+                        {
+                          on: 'hover',
+                          style: {
+                            itemBackground: 'rgba(0, 0, 0, .03)',
+                            itemOpacity: 1,
+                          },
+                        },
```

Comment:
can this be added to the config file?

Reply:
no, the legend is either that block of text or it's [], depending on `legendVisible`